resources/migrations/000_migrations.yaml

```diff
@@ -288,7 +293,14 @@ databaseChangeLog:
               name: name
               type: varchar(254)
           - column:
-              name: rows
+              # liquibase.database.core.H2Database will quote this, because
+              # it is a H2 keyword, which prevents automatic uppercasing by
+              # H2.  So make it uppercase from the start.
+              # An alternative would be to globally set `objectQuotingStrategy`
+              # to `QUOTE_ALL_OBJECTS`, but that will trip the rest of the
+              # Metabase code, because table and column names will then
+              # all be lowercase.
+              name: ROWS
```

Comment:
I think the problem here is that we want this to be `rows` for Postgres and MySQL by `ROWS` for H2